tests/vu/lower/efu.expected

```diff
@@ -0,0 +1,184 @@
+  ERLENG CVF_MAX_EXP: 1f3504f3/+0.00
+  ERLENG CVF_MIN_EXP: 7fffffff/+NaN
+  ERLENG CVF_ONE: 3f13cd3a/+0.58
+  ERLENG CVF_NEGONE: 3f13cd3a/+0.58
+  ERLENG CVF_GARBAGE1: 7fffffff/+NaN
+  ERLENG CVF_GARBAGE2: 1fd9c601/+0.00
+  ERLENG CVF_INCREASING: 3e88d677/+0.27
+  ERLENG CVF_DECREASING: 3e3e26eb/+0.19
+ERSADD:
+  ERSADD CVF_ZERO: 7fffffff/+NaN
+  ERSADD CVF_NEGZERO: 7fffffff/+NaN
+  ERSADD CVF_MAX: 00000000/+0.00
+  ERSADD CVF_MIN: 00000000/+0.00
+  ERSADD CVF_MAX_MANTISSA: 3daaaaac/+0.08
+  ERSADD CVF_MAX_EXP: 00000000/+0.00
+  ERSADD CVF_MIN_EXP: 7fffffff/+NaN
+  ERSADD CVF_ONE: 3eaaaaab/+0.33
+  ERSADD CVF_NEGONE: 3eaaaaab/+0.33
+  ERSADD CVF_GARBAGE1: 7fffffff/+NaN
+  ERSADD CVF_GARBAGE2: 00000000/+0.00
+  ERSADD CVF_INCREASING: 3d924925/+0.07
+  ERSADD CVF_DECREASING: 3d0d3dcb/+0.03
+ERSQRT:
+  ERSQRT CVF_ZERO: 7fffffff/+NaN
+  ERSQRT CVF_NEGZERO: 7fffffff/+NaN
+  ERSQRT CVF_MAX: 1f3504f3/+0.00
+  ERSQRT CVF_MIN: 1f3504f3/+0.00
+  ERSQRT CVF_MAX_MANTISSA: 3f3504f3/+0.71
+  ERSQRT CVF_MAX_EXP: 1f800000/+0.00
+  ERSQRT CVF_MIN_EXP: 7fffffff/+NaN
+  ERSQRT CVF_ONE: 3f800000/+1.00
+  ERSQRT CVF_NEGONE: 3f800000/+1.00
+  ERSQRT CVF_GARBAGE1: 7fffffff/+NaN
+  ERSQRT CVF_GARBAGE2: 2fdbbaac/+0.00
+  ERSQRT CVF_INCREASING: 3f13cd3a/+0.58
+  ERSQRT CVF_DECREASING: 3f3504f3/+0.71
+ESADD:
+  ESADD CVF_ZERO: 00000000/+0.00
+  ESADD CVF_NEGZERO: 00000000/+0.00
+  ESADD CVF_MAX: 7fffffff/+NaN
+  ESADD CVF_MIN: 7fffffff/+NaN
+  ESADD CVF_MAX_MANTISSA: 413ffffe/+12.00
+  ESADD CVF_MAX_EXP: 7fffffff/+NaN
+  ESADD CVF_MIN_EXP: 00000000/+0.00
+  ESADD CVF_ONE: 40400000/+3.00
+  ESADD CVF_NEGONE: 40400000/+3.00
+  ESADD CVF_GARBAGE1: 00000000/+0.00
+  ESADD CVF_GARBAGE2: 7eb0e16b/+117557282134571560288112383319651385344.00
+  ESADD CVF_INCREASING: 41600000/+14.00
+  ESADD CVF_DECREASING: 41e80000/+29.00
+ESIN:
+  ESIN CVF_ZERO: 00000000/+0.00
+  ESIN CVF_NEGZERO: 00000000/+0.00
+  ESIN CVF_MAX: 7fd76aa2/+NaN
+  ESIN CVF_MIN: ffd76aa2/-NaN
+  ESIN CVF_MAX_MANTISSA: 3f68c816/+0.91
+  ESIN CVF_MAX_EXP: 7f2ed54b/+232393151678163554045421220250750287872.00
+  ESIN CVF_MIN_EXP: 00000000/+0.00
+  ESIN CVF_ONE: 3f576aa4/+0.84
+  ESIN CVF_NEGONE: bf576aa4/-0.84
+  ESIN CVF_GARBAGE1: 00000000/+0.00
+  ESIN CVF_GARBAGE2: 7ea2556c/+107889235525184513069623811128269209600.00
+  ESIN CVF_INCREASING: 3e126230/+0.14
+  ESIN CVF_DECREASING: 3f68c815/+0.91
+ESQRT:
+  ESQRT CVF_ZERO: 00000000/+0.00
+  ESQRT CVF_NEGZERO: 00000000/+0.00
+  ESQRT CVF_MAX: 5fb504f3/+26087635204194697216.00
+  ESQRT CVF_MIN: 5fb504f3/+26087635204194697216.00
+  ESQRT CVF_MAX_MANTISSA: 3fb504f3/+1.41
+  ESQRT CVF_MAX_EXP: 5f800000/+18446744073709551616.00
+  ESQRT CVF_MIN_EXP: 00000000/+0.00
+  ESQRT CVF_ONE: 3f800000/+1.00
+  ESQRT CVF_NEGONE: 3f800000/+1.00
+  ESQRT CVF_GARBAGE1: 00000000/+0.00
+  ESQRT CVF_GARBAGE2: 4f152108/+2501969920.00
+  ESQRT CVF_INCREASING: 3fddb3d7/+1.73
+  ESQRT CVF_DECREASING: 3fb504f3/+1.41
+ESUM:
+  ESUM CVF_ZERO: 00000000/+0.00
+  ESUM CVF_NEGZERO: 80000000/-0.00
```

Comment:
This is also a notable result.  4 negatives make a positive, so this seems to have retained the negativeness.  It might be interesting to see if it takes the sign of the first/last field, or if e.g. `-0 + -0 + +0 + -0` would become positive.

-[Unknown]